modules/installation-azure-regions.adoc

```diff
@@ -5,7 +5,7 @@
 [id="installation-azure-regions_{context}"]
 = Supported Azure regions
 
-The installation program dynamically generates the list of available Microsoft Azure regions based on your subscription. The following Azure regions were tested and validated in {product-title} version 4.2.0:
+The installation program dynamically generates the list of available Microsoft Azure regions based on your subscription. The following Azure regions were tested and validated in {product-title} version 4.3.0:
```

Comment:
@e-tienne, is this list still valid for 4.3?

Reply:
@e-tienne Can you please confirm? Thanks!